site/public/js/simple-grading.js

```diff
@@ -658,10 +661,19 @@ function setupSimpleGrading(action) {
             else if (direction == "left") new_selector_array[3] -= 1;
             else if (direction == "right") new_selector_array[3] += 1;
 
-            // TODO: cannot move to NULL section (all cells with id 'cell--#-#')
             if (new_selector_array[2] < 0 && direction == "up") {
-                // // Selection needs to move to above section, if one exists
                 new_selector_array[2] += 1;
+                // // Selection needs to move to above the null section
+                if (new_selector_array[1] == "") {
```

Comment:
It is generally better to use `===` instead of `==` in JS to prevent type inferences from causing problems
```suggestion
                if (new_selector_array[1] === "") {
```